umh-core/pkg/fsmv2/cpu/cpu_metrics_test.go

```
@@ -0,0 +1,169 @@
+		flags := map[deps.GaugeName]bool{
+			deps.GaugeCPUUsageRingActive:       status.Details.UsageRingActive,
+			deps.GaugeCPUHostBusyRingActive:    status.Details.HostBusyRingActive,
+			deps.GaugeCPUHostHeadroomAvailable: status.Details.HostHeadroomAvailable,
+			deps.GaugeCPUThrottleSignalReady:   status.Details.ThrottleSignalReady,
+			deps.GaugeCPUPressureSignalReady:   status.Details.PressureSignalReady,
+		}
+
+		// An implementation that wrote 1 everywhere, or 0 everywhere, would pass
+		// the per-flag assertions below if this tick happened to be unanimous.
+		var trues, falses int
+		for _, v := range flags {
```

golangci-lint (2.13.2)

[error] missing whitespace above this line (no shared variables above range)

(wsl_v5)